src/components/Footer/sgds-footer.ts

```diff
@@ -72,8 +111,8 @@ export class SgdsFooter extends SgdsElement {
               <div class="col">
                 <div class="d-flex justify-content-lg-end">
                   <ul>
-                    <li><a href="${this.contactHref}">Contact</a></li>
-                    <li><a href="${this.feedbackHref}">Feedback</a></li>
+                    <li><a href=${this.contactHref ? this.contactHref : "#"}>Contact</a></li>
```

Comment:
actually putting a default value to contactHref and this line 114 statement essentially is the same effect. but this is okay too